module-types/Cisco/C9400-LC-48H.yaml

```diff
@@ -0,0 +1,201 @@
+---
+manufacturer: Cisco
+part_number: C9400-LC-48H
+model: C9400-LC-48H
+description: Cisco Catalyst 9400 Series 48-Port UPOE+ 10/100/1000 (RJ-45) Line Card
+comments: '[Catalyst 9400 Series Switch Line Cards Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-9400-series-switches/nb-06-cat9400-series-line-data-sheet-cte-en.html)'
+weight: 3.85
+weight_unit: kg
+interfaces:
+  - name: GigabitEthernet{module}/0/1
+    type: 1000base-t
+    poe_mode: pse
+    poe_type: type2-ieee802.3at
```

Comment:
The spec sheet shows that this linecard supports 90W POE++ therefore you should be using `poe_type: type4-ieee802.3bt`